File: settlement-service/src/main/java/com/fastcampuspay/settlement/application/service/RechargeMoney.java
package com.fastcampuspay.settlement.application.service;


import com.fastcampuspay.common.UseCase;
import com.fastcampuspay.settlement.application.port.in.RechargeMoneyCommand;
import com.fastcampuspay.settlement.application.port.in.RechargeMoneyUseCase;
import jakarta.transaction.Transactional;
import lombok.RequiredArgsConstructor;



@RequiredArgsConstructor
@UseCase
@Transactional
public class RechargeMoney implements RechargeMoneyUseCase {

	@Override
	public void rechargeMoney(RechargeMoneyCommand command) {

	}
}

File: settlement-service/src/main/kotlin/com/hmpay/settlement/application/service/RechargeMoney.kt
package com.hmpay.settlement.application.service

import com.hmpay.common.UseCase
import com.hmpay.settlement.application.port.`in`.RechargeMoneyCommand
import com.hmpay.settlement.application.port.`in`.RechargeMoneyUseCase
import jakarta.transaction.Transactional

@UseCase
@Transactional
class RechargeMoney : RechargeMoneyUseCase {
    override fun rechargeMoney(command: RechargeMoneyCommand) {
        // TODO: implement settlement recharge logic
    }
}
